### ISSUE TITLE
npm install error with node v12.14.0 and npm 6.13.4

### ISSUE THREAD

**Issue by AlexSapoznikov** [elrumordelaluz/outline-stroke-cli#2]
npm install outline-stroke-cli fails with following error (node v12.14.0, npm 6.13.4):

```info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.6.1/libvips-8.6.1-darwin-x64.tar.gz
prebuild-install WARN install No prebuilt binaries found (target=12.13.1 runtime=node arch=x64 platform=darwin)
  TOUCH Release/obj.target/libvips-cpp.stamp
  CXX(target) Release/obj.target/sharp/src/common.o
In file included from ../src/common.cc:27:
../src/common.h:82:20: error: no member named 'Handle' in namespace 'v8'
  bool HasAttr(v8::Handle<v8::Object> obj, std::string attr);
               ~~~~^
../src/common.h:82:37: error: expected '(' for function-style cast or type construction
  bool HasAttr(v8::Handle<v8::Object> obj, std::string attr);
                          ~~~~~~~~~~^
../src/common.h:82:39: error: use of undeclared identifier 'obj'
  bool HasAttr(v8::Handle<v8::Object> obj, std::string attr);
                                      ^
../src/common.h:82:56: error: expected '(' for function-style cast or type construction
  bool HasAttr(v8::Handle<v8::Object> obj, std::string attr);
                                           ~~~~~~~~~~~ ^
../src/common.h:83:29: error: no member named 'Handle' in namespace 'v8'
  std::string AttrAsStr(v8::Handle<v8::Object> obj, std::string attr);
                        ~~~~^
../src/common.h:83:46: error: expected '(' for function-style cast or type construction
  std::string AttrAsStr(v8::Handle<v8::Object> obj, std::string attr);
                                   ~~~~~~~~~~^
../src/common.h:83:48: error: use of undeclared identifier 'obj'
  std::string AttrAsStr(v8::Handle<v8::Object> obj, std::string attr);
                                               ^
../src/common.h:83:65: error: expected '(' for function-style cast or type construction
  std::string AttrAsStr(v8::Handle<v8::Object> obj, std::string attr);
                                                    ~~~~~~~~~~~ ^
../src/common.h:84:37: warning: variable templates are a C++14 extension [-Wc++14-extensions]
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                    ^
../src/common.h:84:48: error: no member named 'Handle' in namespace 'v8'
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                           ~~~~^
../src/common.h:84:65: error: expected '(' for function-style cast or type construction
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                                      ~~~~~~~~~~^
../src/common.h:84:67: error: use of undeclared identifier 'obj'
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                                                  ^
../src/common.h:84:84: error: expected '(' for function-style cast or type construction
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                                                       ~~~~~~~~~~~ ^
../src/common.h:84:89: error: expected ';' at end of declaration
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                                                                        ^
                                                                                        ;
../src/common.h:84:90: error: expected unqualified-id
  template<typename T> v8::Local<T> AttrAs(v8::Handle<v8::Object> obj, std::string attr) {
                                                                                         ^
../src/common.h:87:26: warning: variable templates are a C++14 extension [-Wc++14-extensions]
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                         ^
../src/common.h:87:37: error: no member named 'Handle' in namespace 'v8'
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                                ~~~~^
../src/common.h:87:54: error: expected '(' for function-style cast or type construction
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                                           ~~~~~~~~~~^
../src/common.h:87:56: error: use of undeclared identifier 'obj'
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                                                       ^
../src/common.h:87:73: error: expected '(' for function-style cast or type construction
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                                                            ~~~~~~~~~~~ ^
../src/common.h:87:78: error: expected ';' at end of declaration
  template<typename T> T AttrTo(v8::Handle<v8::Object> obj, std::string attr) {
                                                                             ^
                                                                             ;
fatal error: too many errors emitted, stopping now [-ferror-limit=]
2 warnings and 20 errors generated.
make: *** [Release/obj.target/sharp/src/common.o] Error 1
```

Is there something I can do to fix this?

**Comment by AlexSapoznikov** [comment]
Also the same happens to your other library: https://github.com/elrumordelaluz/outline-stroke#readme

**Comment by elrumordelaluz** [comment]
Hi @AlexSapoznikov, thanks for open the _Issue_.

Probably the reason is related to [this](https://github.com/lovell/sharp/issues/1464)

**Comment by elrumordelaluz** [comment]
Solved in `outline-stroke-cli@1.1.1`. Feel free to re-open or open another _Issue_ in any case.
